trino/trino.go

```diff
@@ -114,6 +121,19 @@ const (
 	SSLCertPathConfig        = "SSLCertPath"
 )
 
+var (
+	headersMap = map[string]string{
```

Comment:
nit: rename to `responseToRequestHeaderMap`?

Reply:
Done!